### DISCUSSION TITLE
How to display progress bar programmatically in Nuxt 3 like $nuxt.$loading.start()

### DISCUSSION THREAD

**Discussion by N-Laboratory** [nuxt/nuxt#18885] [Questions]
In Nuxt2, the loading bar can be programmatically started in your components by calling this.$nuxt.$loading.start() to start the loading bar and this.$nuxt.$loading.finish() to finish it.
https://nuxtjs.org/docs/features/loading/#programmatically-starting-the-loading-bar

But loading is not supported in Nuxt 3.
https://nuxt.com/docs/migration/component-options#loading

What should I do in Nuxt3 to display progress bar programmatically?

**Comment by TheAlexLichter** [comment]
See https://github.com/nuxt/nuxt/discussions/16351 for more info.
